R/archive.R

```diff
@@ -25,7 +25,7 @@
 #' @noRd
 validate_version_bound <- function(version_bound, x, na_ok = FALSE,
                                    version_bound_arg = rlang::caller_arg(version_bound),
-                                   x_arg = rlang::caller_arg(version_bound)) {
+                                   x_arg = rlang::caller_arg(x)) {
```

Comment:
This argument appears unused in this function?

Reply:
Right, I'm not sure why; haven't looked enough at function to know if it should be.  But that's also why I felt confident enough to just make this typo fix in case we do use/read it in the future.

Reply:
If you know whether it should be used maybe we could replace this PR with an actual fix.

Reply:
I don't know, let's just go ahead with the small fix for now.